readme.md

```diff
@@ -56,6 +56,7 @@ Type: `object`
 Type: `object`
 
 HTML attributes to add to the link.
+By default there is the `rel="noreferrer"` based on [Google Lighthouse Audits](https://developers.google.com/web/tools/lighthouse/audits/noopener) this should be the default for cross-origin destinations.
```

Comment:
I think this could be written better. And this should also succinctly explain *why* it's the default. You also need to include it in index.d.ts which should be in sync with the readme.